docs/features/auth/certificate.md

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
Given the configuration of the attributes are the same, but the way they are applied not I would maybe reshuffle the docs and promote the configurate to a dedicated section and point to it from the 2 attributes/filters.

Would that make sense?

Reply:
Not exactly, '2 attributes'? The registration of the validator is used in both sections; but otherwise it's different?

Reply:
You are right, sorry about that!